.github/stale.yml

```diff
@@ -0,0 +1,49 @@
+# Configuration for probot-stale - https://github.com/probot/stale
+
+# Number of days of inactivity before an Issue or Pull Request becomes stale
+daysUntilStale: 90
+
+# Number of days of inactivity before a stale Issue or Pull Request is closed.
+# Set to false to disable. If disabled, issues still need to be closed manually, but will remain marked as stale.
```

Comment:
How about enabling only this setting? @blag @Kami 
Tag 90d issues as `stale`, but *don't* close them.

Additional bot comment for 90-days old issues could also bring some life to some stale Issues as Github notifications will come to all st2 contributors and maintainers.

That could be our chance to review the Issue, Re-tag or close it consciously as part of maintenance.

Reply:
I think that's a good compromise.

Reply:
Something like this should work - acd460f5f463bae4a154b0e9ace6b3f0a1f8cc6e. What do you think, @armab?